data/transactions/logic/doc.go

```diff
@@ -125,14 +155,27 @@ func OpImmediateNote(opName string) string {
+	{"bz", "See `bnz` for details on how branches work."},
+	{"b", "See `bnz` for details on how branches work. `b` always jumps to the offset."},
 	{"intcblock", "`intcblock` loads following program bytes into an array of integer constants in the evaluator. These integer constants can be referred to by `intc` and `intc_*` which will push the value onto the stack. Subsequent calls to `intcblock` reset and replace the integer constants available to the script."},
 	{"bytecblock", "`bytecblock` loads the following program bytes into an array of byte string constants in the evaluator. These constants can be referred to by `bytec` and `bytec_*` which will push the value onto the stack. Subsequent calls to `bytecblock` reset and replace the bytes constants available to the script."},
 	{"*", "Overflow is an error condition which halts execution and fails the transaction. Full precision is available from `mulw`."},
 	{"txn", "FirstValidTime causes the program to fail. The field is reserved for future use."},
-	{"gtxn", "for notes on transaction fields available, see `txn`. If this transaction is _i_ in the group, `gtxn i field` is equivalent to `txn field`"},
-	{"btoi", "`btoi` panics if the input is longer than 8 bytes"},
+	{"gtxn", "for notes on transaction fields available, see `txn`. If this transaction is _i_ in the group, `gtxn i field` is equivalent to `txn field`."},
+	{"btoi", "`btoi` panics if the input is longer than 8 bytes."},
+	{"concat", "`concat` panics if the result would be greater than 4096 bytes."},
+	{"app_opted_in", "params: account index, application id (top of the stack on opcode entry)."},
+	{"app_local_get", "params: account index, state key. Return: value. The value is zero if the key does ont exist."},
+	{"app_local_get_ex", "params: account index, application id, state key. Return: did_exist flag (top of the stack), value."},
+	{"app_global_get_ex", "params: application id, state key. Return: value."},
+	{"app_global_get", "params: state key. Return: value. The value is zero if the key does ont exist."},
+	{"app_local_put", "params: account index, state key, value."},
+	{"app_local_del", "params: account index, state key."},
+	{"app_global_del", "params: state key."},
+	{"asset_holding_get", "params: account index, asset id. Return: did_exist flag, value."},
+	{"asset_params_get", "params: account index, asset id. Return: did_exist flag, value."},
```

Comment:
Does a value of 1 for `did_exist` mean the asset exists, or the opposite? (It might be a good idea to add a comment saying somewhere that by convention, a "1" returned by an opcode means "true" and a "0" means "false".